draft-meunier-http-message-signatures-directory.md

```diff
@@ -127,6 +129,41 @@ with different validity period. When rotating keys, clients SHOULD:
 
 Servers SHOULD cache the directory contents and refresh upon expiration.
 
+## Binding keys to the directory authority
+
+To ensure the authenticity and integrity of the key material provided by the
+directory, clients **SHOULD** validate the directory's response.
+
+When a directory server provides a key directory over HTTP or HTTPS, it is
+RECOMMENDED that it constructs and includes one HTTP Message Signatures per keys
```

Comment:
I realized too late that [RFC 9421](https://www.rfc-editor.org/rfc/rfc9421#name-creating-the-signature-base) doesn't actually specify a way to sign message _bodies_ - only headers and derived components. 

In Section 7.4.2, they recommend use of the DIGEST header: https://www.rfc-editor.org/rfc/rfc9421#security-message-content

We should maned this so that `Content-Digest` is generated and included in the response, with the signature components covering `Content-Digest`

Reply:
The change in this PR does not mention signing the digest. Solely `@authority`.

Reply:
The reasoning about not having digest: authority+signature is sufficient to validate one has the correct public key. Digest over the whole body may prevent mirroring.

Should we recommended Content-Digest to be set: maybe. But that would be a separate issue